### ISSUE TITLE
re-frisk warning with latest reagent version. 

### ISSUE THREAD

**Issue by zendevil** [flexsurfer/re-frisk#56]
Use re-frisk with reagent 1.0.0-alpha2

Get the following error

```
------ WARNING #1 - :fn-arity --------------------------------------------------
 Resource: re_frisk/trace.cljs:27:11
--------------------------------------------------------------------------------
  24 |         {:op-type   :render
  25 |          :operation (operation-name c)}
  26 |         (if util/*non-reactive*
  27 |           (reagent.impl.component/do-render c)
-----------------^--------------------------------------------------------------
 Wrong number of args (1) passed to reagent.impl.component/do-render
--------------------------------------------------------------------------------
  28 |           (let [rat        (gob/get c "cljsRatom")
  29 |                 _          (batch/mark-rendered c)
  30 |                 res        (if (nil? rat)
  31 |                              (ratom/run-in-reaction #(reagent.impl.component/do-render c) c "cljsRatom"
--------------------------------------------------------------------------------

------ WARNING #2 - :fn-arity --------------------------------------------------
 Resource: re_frisk/trace.cljs:31:54
--------------------------------------------------------------------------------
  28 |           (let [rat        (gob/get c "cljsRatom")
  29 |                 _          (batch/mark-rendered c)
  30 |                 res        (if (nil? rat)
  31 |                              (ratom/run-in-reaction #(reagent.impl.component/do-render c) c "cljsRatom"
------------------------------------------------------------^-------------------
 Wrong number of args (1) passed to reagent.impl.component/do-render
--------------------------------------------------------------------------------
  32 |                                                     batch/queue-render reagent.impl.component/rat-opts)
  33 |                              (._run rat false))
  34 |                 cljs-ratom (gob/get c "cljsRatom")]         ;; actually a reaction
  35 |             (re-frame.trace/merge-trace!
--------------------------------------------------------------------------------

```

**Comment by Aitem** [comment]
The same problem

**Comment by flexsurfer** [comment]
yeah thanks,  that's because of Monkeypatched render for tracing views, and this recent change in reagent https://github.com/reagent-project/reagent/commit/28c9b8921df58f0cca29bf2ac1e1f4b1e8c90bc4 , so i need to take a look at this

**Comment by Karkunow** [comment]
+1, have the same problem

**Comment by flexsurfer** [comment]
fixed in 1.3.4
